src/preact/component/component.type.js

```diff
@@ -53,3 +53,32 @@ var RendererFunctionResponseType;
  * @typedef {function(!JsonObject):(?RendererFunctionResponseType|!Promise<?RendererFunctionResponseType>)}
  */
 var RendererFunctionType;
+
+/** @const */
+var IframeEmbedDef = {};
+
+/**
+ * @typedef {{
+ *   allow: (string|undefined),
+ *   allowFullScreen: (boolean|undefined),
+ *   allowTransparency: (boolean|undefined),
+ *   loading: (Loading),
+ *   manageMessageHandler: (function({current: ?}, function):function|undefined),
```

Comment:
`ref`'s type is `HTMLIFrameElement`, right?

Reply:
yes, updated